lib/puppet/parser/functions/bracketize.rb

```diff
@@ -8,6 +8,8 @@ module Puppet::Parser::Functions
     @return [Variant[String, Array[String]]]
     ENDHEREDOC
 
+function_simplib_deprecation(['bracketize', 'bracketize is deprecated, please use tcpwrappers::bracketize'])
```

Comment:
```suggestion
function_simplib_deprecation(['bracketize', 'bracketize is deprecated'])
```

Reply:
thanks for that catch, forgot to go back to that after one of our talks.